src/services/videoconference/index.js

```diff
@@ -470,6 +478,14 @@ class CreateVideoconferenceService extends VideoconferenceBaseService {
 					settings,
 					true,
 				);
+
+				// permissionScopeName enum = ['courses','teams']
+				await this.app.service(`/${permissionScopeName}/`).get(scopeId, {
+					query: { $populate: ['userIds'], },
```

Comment:
here, add `teacherIds` and `substitutionIds` for courses, for teams we have teamUserSchema in userIds having the userIds. because of that we may use the team and course members service instead